tools/pv_tree_generator/build_tree_test.py

```diff
@@ -103,10 +103,9 @@ def test_search_white_list(self, mock_get_sv, mock_get_triples):
         root_search = build_tree.build_tree(vertical, pop_obs_spec[vertical],
             stat_vars, True)
         data[1][vertical] = root
-        # 3 from pos + 4 from starvars
-        self.assertEqual(data[0]['Demographics']['count'], 7) 
-        self.assertEqual(data[1]['Demographics']['count'], 7)
-        self.assertEqual(data[1]['Demographics']['search_count'], 5)
+        self.assertEqual(data[0]['Demographics']['count'], 4) 
+        self.assertEqual(data[1]['Demographics']['count'], 4)
+        self.assertEqual(data[1]['Demographics']['search_count'], 2)
```

Comment:
can also add test for the types. Ideally different UI nodes have different place types

Reply:
done.